### ISSUE TITLE
Recruitment

### ISSUE THREAD

**Issue by rebeccaradding** [foundersandcoders/nazareth#13]
Recruiting candidates for Nazareth has been challenging (for several reasons, including perhaps #11). 

**Promotional efforts thus far include:**

- Informational posts on our Facebook page: reviews, alumni stories, etc.
- Facebook ads promoting the Founders & Coders two minute video 
- Facebook ads promoting the Founders & Coders "Nazareth" 30-second video
- Asking friends and partners to share and repost our Facebook posts about the programme
- Distributing an English-language application announcement to all of our contacts in the region #1 as well as other community organizations
- Distributing the English announcement to Israeli college and university alumni networks and computer science departments
- Creating a Coding for Everyone - Nazareth channel on Gitter
- Releasing an Arabic language application announcement on Bokra.net (who then distributed it to several smaller sites)
- Collaborating with Hasoub to advertise the programme and creating a Facebook group for interested applicants based in Israel
- Filming a video and conducting an interview with Bokra.net
- Hosting a meet-up in Nazareth for prospective applicants on 12/8
- Collaborating with the UK Israel Tech Hub, who have promoted the programme on their website, Facebook page, and in their newsletter
- Promoting the programme on the nazarethinfo.org page for the Nazareth Cultural & Tourism Association
- Contacting journalists in Israel about the application - leading to an article in the Jerusalem Post

---
 
Given that the application deadline is just days away, it seems the best use of our time at this point would be to contact people who have already expressed interest to find out:

- Whether they are still considering the programme
- Whether they would be more likely to participate in a 16-week programme
- What, if any, prerequisites they have done (or what other preparation they have for the programme based on previous studies)
- Whether they would be interested in the meet-ups planned for January/February as a way to test out Founders & Coders
- Whether they have any other questions about the programme

In the Expression of Interest sheet, I have created a separate tab containing only local and international candidates who have expressed interest in Nazareth.

https://docs.google.com/spreadsheets/d/173H7GQL1V9JPZgaXX35P0UTrPlkviqFCo1pjAJRaDII/edit#gid=1465055861

When I brought this up last week, both @shireenzoaby and @esraajb indicated a willingness to contact these potential candidates. We agreed that conducting this _user research_ will likely provide us with valuable insight that will help us determine both where we stand and what our next steps could be.

I propose that @shireenzoaby and @esraajb **make contact** with each of the people on the list before **December 30** and _record_ what you learn from the conversations in the spreadsheet in a **notes** column (or some other form that works for both of you). @des-des, it might make sense for you to contact the international applicants (either by e-mail, Facebook, or Gitter).

I'm aware that each of you has already spoken with many of them. To start, I suggest:

1. By 12/23 (Friday), you both go through the list and make notes about people you've already spoken to.
1. By 12/24, you determine whether/how to divide up the list (including if @des-des covers international applicants)
1. By 12/30, you record the results of your contact with each of the candidates

I will be mostly offline beginning 12/22 through the first week in January, so please let me know what you think about this approach as soon as you can. Thanks!

**Comment by esraajb** [comment]
i went through the expressions of interest, 
must of them dont have FCC and CW accounts. 
do you think there is a point of contacting people who didnt make any progress ?

**Comment by rebeccaradding** [comment]
@esraajb yes, because you'll find out why. They may have some kind of technical background, they may have unanswered questions about the programme - you won't know until you ask.

**Comment by rebeccaradding** [comment]
Given uncertainty around participation of international applicants, I would also suggest that @sofer and/or @des-des make phone calls to the London applicants who have been offered the opportunity to apply to Nazareth. Direct contact will give us a much clearer picture of how many are interested and what might be holding them back.

What do you think?
